### PR TITLE
fix(codebases): prevent new tab opening when creating workspace via d…

### PR DESCRIPTION
**Pull request by aptmac**
…ropdown menu

This PR addresses issue #3995 [[0]](https://github.com/openshiftio/openshift.io/issues/3995), in which creating a workspace via the dropdown menu on the codebases page will still try to open a new tab.

The behaviour of opening and creating workspaces was tweaked in a recent PR that addressed ad-block preventing the opening of new tabs [[1]](https://github.com/openshiftio/openshift.io/issues/3565). During review, it was discussed that separating the creation & opening logic could be advantageous with regards to getting around ad-blockers [[2]](https://github.com/fabric8-ui/fabric8-ui/pull/2972#issue-191572823), but I missed tweaking the drop-down menu action.

[0] https://github.com/openshiftio/openshift.io/issues/3995
[1] https://github.com/openshiftio/openshift.io/issues/3565
[2] https://github.com/fabric8-ui/fabric8-ui/pull/2972#issue-191572823

### CODE REVIEW

File: src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts [hunk 1]
```diff
@@ -62,7 +59,6 @@ export class CodebasesItemActionsComponent implements OnDestroy, OnInit {
             this.workspaceBusy = false;
             if (workspaceLinks != undefined) {
               let name = this.getWorkspaceName(workspaceLinks.links.open);
-              this.windowService.open(workspaceLinks.links.open, name);
```
Comment (joshuawilson) [inline src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts]: How does it open if you remove this line? Am I missing it someplace else?

Comment (aptmac) [reply src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts]: It opens when the open button is hit on the `codebases-item-workspaces` component [0]

https://github.com/aptmac/fabric8-ui/blob/c2e016d124b0a13174d21419c44864a209ec2266/src/app/space/create/codebases/codebases-item-workspaces/codebases-item-workspaces.component.ts#L126

Comment (joshuawilson) [reply src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts]: so what did the open you remove do? open it a 2nd time?

Comment (aptmac) [reply src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts]: No, this open triggers immediately after workspace creation, which causes it to hit the ad-blocker if present.

This modified behaviour mirrors what was changed in the `codebases-item-workspaces` component in [[1]](https://github.com/fabric8-ui/fabric8-ui/pull/2972), in which the creation and open logic were decoupled.

[1] https://github.com/fabric8-ui/fabric8-ui/pull/2972

Comment (joshuawilson) [reply src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts]: ok

**Pull request merged**